poco_vendorConfig.cmake.in

```diff
@@ -6,7 +6,7 @@ set(_@PROJECT_NAME@_CONFIG_INCLUDED TRUE)
 
 # output package information
 if(NOT @PROJECT_NAME@_FIND_QUIETLY)
-  message(" - Found @PROJECT_NAME@: @PACKAGE_VERSION@ (${@PROJECT_NAME@_DIR})")
+  message(STATUS " - Found @PROJECT_NAME@: @PACKAGE_VERSION@ (${@PROJECT_NAME@_DIR})")
```

Comment:
Since `STATUS` will already print a leading `- ` can you please remove ` - ` from the message.